src/Apps/ArtQuiz/Components/ArtQuizArtworksCardMetadata.tsx

```diff
@@ -1,14 +1,14 @@
-import { Box, BoxProps, Text } from "@artsy/palette"
+import { Box, type BoxProps, Text } from "@artsy/palette"
```

Comment:
interesting, I hadn't seen this in TS before

Reply:
New(ish) js pattern. Starting to see it more and more as a default